### DISCUSSION TITLE
The preprocessing transforms in train.json and inference.json do not match for whole body ct segmentation bundle

### DISCUSSION THREAD

**Discussion by mpsampat** [Project-MONAI/model-zoo#487] [General]
Hi All! 

i am trying to convert whole body ct segmentation bundle into a monai deploy MAP. 
I dont want to use the MONAIBundleInferenceOperator  since
we have a very strong preference for monai deploy MAP over bundles.  

I noticed the pre-processing transforms in train.json and inference.json for whole body ct segmentation bundle do not match. 
Inference.json  has both:  
  {
                "_target_": "NormalizeIntensityd",
                "keys": "image",
                "nonzero": true
            },
            {
                "_target_": "ScaleIntensityd",
                "keys": "image",
                "minv": -1.0,
                "maxv": 1.0
            }
while train.json has NormalizeIntensityd but not ScaleIntensityd
{
                "_target_": "NormalizeIntensityd",
                "keys": "image",
                "nonzero": true
            },
Could someone please check it and let me know ? 
I can then create an operator for whole body ct segmentation and submit it to monai deploy app github

**Comment by yiheng-wang-nv** [comment]
HI @tangy5 , could you please help to clarify the difference here?
cc @KumoLiu @wyli @Nic-Ma

**Comment by KumoLiu** [comment]
Hi @mpsampat, it seems there also have a `ScaleIntensityd` in the "inference.json".
https://github.com/Project-MONAI/model-zoo/blob/99b8be72b8dc5602f7cfa3e0d140ebfa74e1b664/models/wholeBody_ct_segmentation/configs/inference.json#L70

I hope it helps, thanks!

**Comment by mpsampat** [comment]
thanks all
I was able to use the ConfigParser and confirm it is there in the pre-processing steps. the code is below: 
parser  = ConfigParser()
parser.read_config('configs/inference.json')
preprocessing = parser.get_parsed_content("preprocessing")

# now we can check the list of transforms as follows: 
preprocessing.transforms
(<monai.transforms.io.dictionary.LoadImaged at 0x7f7d5a84cbe0>,
 <monai.transforms.utility.dictionary.EnsureTyped at 0x7f7d5ae43f40>,
 <monai.transforms.utility.dictionary.EnsureChannelFirstd at 0x7f7d5ae43fd0>,
 <monai.transforms.spatial.dictionary.Orientationd at 0x7f7d5ae43430>,
 <monai.transforms.spatial.dictionary.Spacingd at 0x7f7d5ae43d30>,
 <monai.transforms.intensity.dictionary.NormalizeIntensityd at 0x7f7d5ae43820>,
 <monai.transforms.intensity.dictionary.ScaleIntensityd at 0x7f7d5a35efa0>)
